src/DataLayer/CmuData.h

```diff
@@ -1,14 +1,14 @@
 #pragma once
-/*-------------------------------------------------------
-   By Jordan Heinrichs for the Solar Car Team
-   Copyright (c) 2014 by University of Calgary Solar Car Team
--------------------------------------------------------*/
 
 struct CmuData
 {
-   CmuData();
+   CmuData()
+   : cellVoltage{63, 64, 65, 66, 67, 68, 69, 70}
+   , pcbTemperature(47)
+   , cellTemperature{48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62}
+   {}
 
-   float pcbTemperature;
-   float cellTemperature;
-   float cellVoltage[8];
+   unsigned short cellVoltage[8];

```

Comment:
Created new issues for new recruits, make data variable for CMUs, MPPTs, and MotorDetails.